packages/experimental/Shadow/src/__tests__/Shadow.test.tsx

```diff
@@ -104,6 +120,21 @@ describe('Shadow component tests', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  it('Shadow on a child with margin and padding', () => {
+    const tree = renderer.create(<TestShadowOnChildViewWithProps childViewStyleProps={{ margin: 2, padding: 2 }} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
Also add tests for just margin and just padding?

Reply:
Personally I think it's more obvious when something is wrong when there's both margin/padding set on the same view. Are there any specific things you think having specific tests will catch?

Reply:
I always get the two confused... So I kinda wanted to see a test with one or the other 😅